modules/docs/src/Volo.Docs.Admin.Web/Pages/Docs/Admin/Documents/index.js

```diff
@@ -51,14 +51,10 @@
                                 visible: abp.auth.isGranted(
                                     'Docs.Admin.Documents'
                                 ),
-                                confirmMessage: function (data) {
-                                    return l('RemoveFromCacheConfirmation');
-                                },
                                 action: function (data) {
                                     service
                                         .removeFromCache(data.record.id)
                                         .then(function () {
-                                            abp.message.success(l('RemovedFromCache'));
```

Comment:
I [said](https://github.com/volosoft/volo/issues/4538) to show notification instead of alert. `abp.notify.success(l('RemovedFromCache'));`